vision_evaluation/evaluators.py

```diff
@@ -13,6 +13,13 @@
 from functools import reduce
 
 
+def _indices_to_vec(indices, length):
+    target_vec = np.zeros(length, dtype=int)
+    target_vec[indices] = 1
+
+    return target_vec
+
+
 def _targets_to_mat(targets, n_class):
     if len(targets.shape) == 1:
         target_mat = np.zeros((len(targets), n_class), dtype=int)
```

Comment:
not used anymore?

Reply:
You mean targets_to_mat? It is still needed for other flavors of AP calculation.

It is not needed for macro as we could compute ap for each class in macro case to save memory